packages/react-components/src/group/src/Group.tsx

```diff
@@ -1,8 +1,8 @@
-import { ComponentProps, ElementType, ForwardedRef, ReactNode } from "react";
-import { Flex, useFlexAlignment, useFlexDirection } from "../../layout";
-import { forwardRef, isNil, mergeProps } from "../../shared";
+import { ComponentProps, ElementType, ForwardedRef, ReactNode, forwardRef } from "react";
+import { Flex, FlexProps, useFlexAlignment, useFlexDirection } from "../../layout";
+import { isNil, mergeProps } from "../../shared";
 
-export interface InnerGroupProps {
+export interface InnerGroupProps extends Omit<FlexProps, "wrap">{
```

Comment:
I wouldn't extend Flex, I think it's best to duplicate the props as they could diverge in the future. Also, I always viewed Group as a simplified version of Flex. Flex will most likely change a lot in the "CSS in JS" PR.